_objects.layout.scss

```diff
@@ -47,6 +47,22 @@
  * This will cause the system to fill up from either the centre or the right
  * hand side. Default behaviour is to fill up the layout system from the left.
  *
+ * By default we add the gutter and offset to the left of `.o-layout` and
+ * `o-layout__item`. There are however situations where it is beneficial to
+ * distribute out gutter evenly across the left and right using the
+ * `.o-layout--balanced` modifier.
+ *
+ *   <div class="o-layout  o-layout--balanced">
+ *     <div class="o-layout__item  u-width-1/2">
+ *     </div>
+ *     <div class="o-layout__item  u-width-1/2">
+ *     </div>
+ *   </div>
+ *
+ * This will add half our gutter spacing to both the left and right side. This
+ * can be useful when adding a divider between two or more `.o-layout__item`
+ * elements etc.
```

Comment:
Separate concern but we're probably getting to a point where it would be valuable to stick this documentation in a Markdown file for us to consume in sky.com/toolkit

Reply:
I absolutely agree, will raise a ticket.